src/Workspaces/Core/Portable/CodeGeneration/AbstractCodeGenerationService.cs

```diff
@@ -123,13 +133,18 @@ protected static T Cast<T>(object value)
             where TDeclarationNode2 : SyntaxNode
             where TDeclarationNode3 : SyntaxNode
         {
+            if (destination == null)
```

Comment:
Will do


-          heejae

From: Manish Vasani [mailto:notifications@github.com]
Sent: Wednesday, April 15, 2015 1:00 PM
To: dotnet/roslyn
Cc: HeeJae Chang
Subject: Re: [roslyn] original watson bug is fixed by using extension manager when we do info ... (#2003)


In src/Workspaces/Core/Portable/CodeGeneration/AbstractCodeGenerationService.cs<https://github.com/dotnet/roslyn/pull/2003#discussion_r28456619>:

> @@ -123,13 +133,18 @@ protected static T Cast<T>(object value)

>              where TDeclarationNode2 : SyntaxNode

>              where TDeclarationNode3 : SyntaxNode

>          {

> +            if (destination == null)

Can you please file a bug to track cleaning up entire CodeGen service to handle null arguments for all public entry points? I remember seeing a few more elsewhere.

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/roslyn/pull/2003/files#r28456619>.